@navikt/ds-css/modal/index.css

```diff
@@ -0,0 +1,56 @@
+.ReactModal__Body--open {
+  overflow: hidden;
+}
+.navds-modal {
+  --modal-border-radius: 4px;
+
+  background-color: var(--navds-color-white);
+  display: block;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: var(--modal-border-radius);
+  position: relative;
+  flex-grow: 0;
+  overflow: auto;
+  max-height: 100%;
+  margin-bottom: 0;
+  z-index: var(--navds-z-index-modal-content);
+}
+.navds-modal:focus,
+.navds-modal--focus {
+  box-shadow: var(--navds-shadow-focus);
+  outline: none;
+}
+.navds-modal__overlay {
+  position: fixed;
+  z-index: var(--navds-z-index-modal-overlay);
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: var(--navds-color-modal-overlay);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+}
+.navds-modal__overlay:active
+  .navds-modal:not(:active)
+  .navds-modal__button--shake {
+  transform: rotate(10deg);
+  transition: transform 100ms cubic-bezier(0.82, -2, 1, 0.3);
+}
+.navds-modal__button--shake {
+  transform: rotate(0deg);
+  transition: transform 100ms cubic-bezier(0.82, 2, 1, 0.3);
+}
+
+.navds-modal__button {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+.navds-modal__button svg {
+  /* TODO: Set to 1rem when close icon is fixed to 16x16px */
```

Comment:
Skal vi fikse dette før vi merger?

Reply:
Burde helst være fikset ja, men har en mistanke om at ikonet ikke vil se bra ut hvis man bare gjør strekene lengre for å nå 16x16 px. Får høre med Wei